client/lib/signup/site-type.js

```diff
@@ -9,6 +9,38 @@ import { find, get } from 'lodash';
  * Internal dependencies
  */
 
+const getSiteTypePropertyDefaults = propertyKey =>
+	get(
+		{
+			// General copy
+			siteMockupHelpTipCopy: i18n.translate(
+				'Scroll down to see your site. Once you complete setup you’ll be able to customize it further.'
+			),
+			// Site title step
+			siteTitleLabel: i18n.translate( 'Give your site a name' ),
+			siteTitleSubheader: i18n.translate(
+				'This will appear at the top of your site and can be changed at anytime.'
+			),
+			siteTitlePlaceholder: i18n.translate( 'default siteTitlePlaceholder' ),
+			// Site topic step
+			siteTopicHeader: i18n.translate( 'What is your site about?' ),
+			siteTopicSubheader: i18n.translate(
+				"We'll add relevant content to your site to help you get started."
+			),
+			siteTopicInputPlaceholder: i18n.translate( 'Enter a topic or choose one from below.' ),
+			// Domains step
+			domainsStepHeader: i18n.translate( 'Give your site an address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Give your site an address.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Let's implement that, please?

Reply:
Hi @yoavf  :)

I've been asked specifically to remove the period in each of these. I believe we're going for a standard of no period to end the step headings, just the subheadings.
I can't remember where I saw that specifically so tagging @fditrapani in case he has any extra info.

Here's one of the cards with the copy instructions: 
https://github.com/Automattic/zelda-private/issues/23

Reply:
Oh - Ok - the bot missed that this was a change of an existing string. In this case, we'll keep the existing translations when the string changes, no problem. :)